runelite-client/src/main/java/net/runelite/client/plugins/hamstoreroom/HamOverlay.java

```diff
@@ -46,10 +40,10 @@
 	private final Client client;
 	private final HamPlugin hamPlugin;
 
-	private boolean drawSteel = false;
-	private boolean drawIron = false;
-	private boolean drawSilver = false;
-	private boolean drawBronze = false;
+	public boolean drawSteel = false;
```

Comment:
These should really be in the Plugin

Reply:
Also not public. Use a Lombok `@Getter` annotation with an `AccessLevel.PACKAGE`